package.json

```diff
@@ -8,6 +8,7 @@
   },
   "scripts": {
     "publish": "lerna publish",
+    "precommit": "yarn plop docs",
```

Comment:
@TheMallen what are your thoughts about doing this in a precommit hook? Should we instead just run a test for it in CI? Or just make people run `yarn plop docs` when they think of it?

Reply:
`yarn plop docs` + test is nicer I think